src/ert/gui/model/job_list.py

```diff
@@ -81,22 +80,16 @@ def setSourceModel(self, sourceModel: QAbstractItemModel) -> None:
     @staticmethod
     def headerData(section: int, orientation: Qt.Orientation, role: Qt.UserRole) -> Any:
         if role != Qt.DisplayRole:
-            return QVariant()
+            return None
```

Comment:
Is this related to the issue?

Reply:
We made it more readable while trying to find the problem. Should I have it in a separate commit or drop it?